### DISCUSSION TITLE
AGW R1.4 On baremetal install docs

### DISCUSSION THREAD

**Discussion by pkapatsos-blinq** [magma/magma#6420] [Q&A]
Hi the [docs](https://docs.magmacore.org/docs/lte/setup_deb) for BM install, in step 2 has changed? 
If you look at R1.3 or prior . and am certain R1.4 was as such on its released date in march. , or look at the master docs  step 2 it shows or states to , run a agw_install script to clone and configure magma . However the R1.4 docs now show , to go to a dir and run the script ? Are you expecting we clone magma on the BM server first now?
I don think this is correct , can anyone comment .tks.

**Comment by karthiksubraveti** [comment]
You shouldn't have to go to a dir and run the script. agw install script clones the repo and runs the script. The instructions earlier added were done so to run precheck script separately. 

@tmdzk folded the prechecks into the agw install script.
https://github.com/magma/magma/pull/5000
